### PR TITLE
add libsodium-wrappers v0.2.10

### LINKED ISSUE #5132: [Request] Add libsodium.js
**Library name:** libsodium
**Git repository url:** https://github.com/jedisct1/libsodium.js
**npm package url(optional):** https://github.com/paixaop/node-sodium
**Official homepage:** https://download.libsodium.org/doc/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/jedisct1/libsodium.js
**Watch 8
  Star 104
 Fork 19**
@LeaYeh could u help me check this PR for #5132? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/libsodium-wrappers/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "libsodium-wrappers",
+  "version": "0.2.10",
+  "filename": "modules/libsodium-wrappers.min.js",
```
Comment (LeaYeh) [inline ajax/libs/libsodium-wrappers/package.json]: in [dist/modules](https://github.com/jedisct1/libsodium.js/tree/0.2.10/dist/modules) i didnt find libsodium-wrappers.min.js, maybe you should check it again :p

**Comment by maruilian11** [comment]
@LeaYeh because the `filename` need to point to minified file, so i minify `libsodium-wrappers.js` myself , and named the file `libsodium-wrappers.min.js`, you can see the `requiredFiles` under the package.json, it will remind the maintainer when update the lib to the new version which files are required on cdnjs but not on the original git repo, and `libsodium-wrappers.min.js` is already mention in `filename`, so it doesn't need to be in `requiredFiles`, i forget this thing before, i just revise the `requiredFiles`.
is the explaining ok for you? could you help me check this PR again? thank you!

**Comment by LeaYeh** [comment]
oops sorry for that i didnt fellow discuss about `requiredFiles`, you are right.
@PeterDaveHello  could you please check this pr?

**Comment by PeterDaveHello** [comment]
Do we need `modules`? Or just `browsers/combined`?

**Comment by maruilian11** [comment]
@PeterDaveHello i thought we need `modules` because the author said `dist/modules/libsodium-wrappers.js` is the main file.
https://github.com/jedisct1/libsodium.js/blob/master/package.json

**Comment by PeterDaveHello** [comment]
It's `browser` folder there, `modules` may not be used for browser client.

**Comment by maruilian11** [comment]
@LeaYeh i have removed `modules` folder, could you help me check it again? thank you!

**Comment by LeaYeh** [comment]
i think it is ok. @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Let's make `dist/browsers/combined/` the basePath.

**Comment by PeterDaveHello** [comment]
What's wrong here?

**Comment by maruilian11** [comment]
just keep trying to pass the test = =

**Comment by maruilian11** [comment]
@LeaYeh i have modified `bathPath`, could u help me check out this PR? thank you!

**Comment by LeaYeh** [comment]
ok. @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@maruilian11 do not update your PR again after reviewing!

**Comment by PeterDaveHello** [comment]
Let's make the PR's title shorter in future @cdnjs/intern, leave detail in commit message.

**Comment by Amomo** [comment]
ok~

**Comment by maruilian11** [comment]
@PeterDaveHello because the PR is updated 2 days ago, you said cdnjs/intern should always keep commit rebase with latest master.
btw, could you give an example short title for this PR? thanks.

**Comment by PeterDaveHello** [comment]
Didn't I?

**Comment by maruilian11** [comment]
ok

**Pull request merged**